recipe/meta.yaml

```diff
@@ -1,31 +1,190 @@
+        - test -f $PREFIX/lib/libcblas.{{ version_major }}.dylib   # [osx]
+        - if not exist %LIBRARY_BIN%/libcblas.dll exit 1           # [win]
+
+  {% if blas_impl != 'blis' %}
+  - name: liblapack
+    script: build_pkg.sh     # [unix]
+    script: build_pkg.bat    # [win]
+    build:
+      string: "{{ build_num }}_{{ blas_impl }}"
+      run_exports:
+        - {{ pin_subpackage("liblapack", max_pin="x.x") }}
+      track_features:
+       - blas_{{ blas_impl }}     # [blas_impl != blas_default_impl]
+    requirements:
+      host:
+        - {{ pin_subpackage("libblas", exact=True) }}
+      run:
+        - {{ pin_subpackage("libblas", exact=True) }}
+      run_constrained:
+        - {{ pin_subpackage("libcblas", exact=True) }}
+        - {{ pin_subpackage("liblapacke", exact=True) }}
+    files:
+      - lib/liblapack.so.{{ version_major }}      # [linux]
+      - lib/liblapack.{{ version_major }}.dylib   # [osx]
+      - Library/bin/liblapack.dll                 # [win]
+    test:
+      commands:
+        - test -f $PREFIX/lib/liblapack.so.{{ version_major }}      # [linux]
+        - test -f $PREFIX/lib/liblapack.{{ version_major }}.dylib   # [osx]
+        - if not exist %LIBRARY_BIN%/liblapack.dll exit 1           # [win]
+
+  - name: liblapacke
+    script: build_pkg.sh     # [unix]
+    script: build_pkg.bat    # [win]
+    build:
+      string: "{{ build_num }}_{{ blas_impl }}"
+      run_exports:
+        - {{ pin_subpackage("liblapacke", max_pin="x.x") }}
+      track_features:
+       - blas_{{ blas_impl }}     # [blas_impl != blas_default_impl]
+    requirements:
+      - {{ pin_subpackage("libblas", exact=True) }}
+      - {{ pin_subpackage("libcblas", exact=True) }}
+      - {{ pin_subpackage("liblapack", exact=True) }}
+    files:
+      - lib/liblapacke.so.{{ version_major }}      # [linux]
+      - lib/liblapacke.{{ version_major }}.dylib   # [osx]
+      - Library/bin/liblapacke.dll                 # [win]
+    test:
+      commands:
+        - test -f $PREFIX/lib/liblapacke.so.{{ version_major }}      # [linux]
+        - test -f $PREFIX/lib/liblapacke.{{ version_major }}.dylib   # [osx]
+        - if not exist %LIBRARY_BIN%/liblapacke.dll exit 1           # [win]
+  {% endif %}
 
-test:
-  commands: 
-    - echo 'test'  # [unix]    
+  # For compatiblity
+  - name: blas
+    version: "{{ blas_major }}.{{ build_num }}"
+    script: test_blas.sh   # [unix]
+    script: test_blas.bat  # [win]
+    build:
+      string: "{{ blas_impl }}"
+      activate_in_script: True
+    requirements:
+      build:
+        - {{ compiler('c') }}                 # [unix]
+        - {{ compiler('fortran') }}           # [unix]
+        - {{ compiler('m2w64_c') }}           # [win]
+        - {{ compiler('m2w64_fortran') }}     # [win]
+        - cmake
+        - m2-make   # [win]
+        - posix     # [win]
+      host:
+      # Building with blis fails due to a conda-build bug
```

Comment:
Has this been resolved? By the sound of [this comment](https://github.com/conda-forge/blas-feedstock/pull/16#issuecomment-471365135), it should be fine now? @isuruf